Makefile

```diff
@@ -2,8 +2,8 @@ IMAGE_REGISTRY ?= quay.io
 IMAGE_REPO ?= nmstate
 
 HANDLER_IMAGE_NAME ?= kubernetes-nmstate-handler
-HANDLER_IMAGE_SUFFIX ?=
-HANDLER_IMAGE_FULL_NAME ?= $(IMAGE_REPO)/$(HANDLER_IMAGE_NAME)$(HANDLER_IMAGE_SUFFIX)
+HANDLER_IMAGE_SUFFIX ?= "latest"
+HANDLER_IMAGE_FULL_NAME ?= $(IMAGE_REPO)/$(HANDLER_IMAGE_NAME):$(HANDLER_IMAGE_SUFFIX)
```

Comment:
This was done to be able to do use image without tags since ".../kubernetes-nmstate-handler:" would fail, again if we remove the latest concept from this we don't need this change.